### PR TITLE
Brand root directory 

### PR DESCRIPTION
**Pull request by yoosofan**

### CODE REVIEW

**Comment by yoosofan** [comment]
https://github.com/ombre-projects/ombre/issues/63

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,171 +1,33 @@
...
-(eg, posted to fpaste.org with a long enough timeout and a link
-posted to #monero-dev on irc.freenode.net).
+Patches are preferably to be sent via a github pull request.
 
 Patches should be self contained. A good rule of thumb is to have
 one patch per separate issue, feature, or logical change. Also, no
 other changes, such as random whitespace changes or reindentation.
 Following the code style of the particular chunk of code you're
-modifying is encouraged. Proper squashing should be done (eg, if
+modifying is encourgaged. Proper squashing should be done (eg, if
```
Comment (mrheinen) [inline CONTRIBUTING.md]: encouraged

Comment (yoosofan) [reply CONTRIBUTING.md]: Okay, I did it in my second commit. Also update my fork.

File: CMakeLists.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2014-2017, The Monero Project
```
Comment (mrheinen) [inline CMakeLists.txt]: Can you leave the original date? (also in other places?)

File: CMakeLists.txt [hunk 3]
```diff
@@ -336,7 +337,7 @@ if(SANITIZE)
 endif()
 
 # Set default blockchain storage location:
-# memory was the default in Cryptonote before Monero implemented LMDB, it still works but is unnecessary.
+# memory was the default in Cryptonote before Ombre implemented LMDB, it still works but is unnecessary.
```
Comment (mrheinen) [inline CMakeLists.txt]: lets just remove this line ?

File: CMakeLists_IOS.txt [hunk 4]
```diff
@@ -1,6 +1,8 @@
-# Portions Copyright (c) 2017-2018, The Monero Project
+# Copyright (c) 2017-2018, The Monero Project
```
Comment (mrheinen) [inline CMakeLists_IOS.txt]: Add back "Portions" before the monero copyright line.   Copyright statements that are already in the files should remain untouched. I made a mistake once to change it (I was ignorant) and got served with a DMCA take down o-O

File: LICENSE [hunk 5]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2014-2018, The Monero Project
```
Comment (mrheinen) [inline LICENSE]: Please do not remove the Monero line

File: README.md [hunk 6]
```diff
@@ -1,696 +1,76 @@
...
-#### OpenBSD >= 6.2
-
-You will need to add a few packages to your system. `pkg_add cmake zeromq libiconv`.
-
-The doxygen and graphviz packages are optional and require the xbase set.
-
-
-Build the Boost library using clang. This guide is derived from: https://github.com/bitcoin/bitcoin/blob/master/doc/build-openbsd.md
-
-We assume you are compiling with a non-root user and you have `doas` enabled.
```
Comment (mrheinen) [inline README.md:397]: Do we still have the build instructions somewhere ? I think in the current codebase we have them in a separate file and linked it in the readme.  The build instructions likely need to be updated with those of Monero because the code and dependencies are very different now.

File: CMakeLists_IOS.txt (hunk 4 above)
Comment (yoosofan) [reply CMakeLists_IOS.txt]: Okay I will do it. Sorry I did not know.

File: README.md (hunk 6 above)
Comment (yoosofan) [reply README.md:397]: No, we do not have. It should be added. In Ubuntu like 
```
sudo apt install -y build-essential cmake libboost-all-dev miniupnpc libunbound-dev graphviz doxygen libunwind8-dev pkg-config libssl-dev libcurl4-openssl-dev libgtest-dev libreadline-dev libminiupnpc-dev libzmq3-dev libsodium-dev libsodium18 libsodium-dbg 
```

File: LICENSE (hunk 5 above)
Comment (yoosofan) [reply LICENSE]: Sorry I did not know. I corrected it.

File: CMakeLists.txt (hunk 2 above)
Comment (yoosofan) [reply CMakeLists.txt]: Yes, I do.

File: CMakeLists.txt (hunk 3 above)
Comment (yoosofan) [reply CMakeLists.txt]: Done

File: CMakeLists_IOS.txt [hunk 7]
```diff
@@ -1,6 +1,8 @@
-# Portions Copyright (c) 2017-2018, The Monero Project
+# Copyright (c) 2017, The Monero Project
```
Comment (mrheinen) [inline CMakeLists_IOS.txt]: 2017-2018

Comment (mrheinen) [reply CMakeLists_IOS.txt]: Portions Copyright (c) 2017-2018, The Monero Project

File: LICENSE [hunk 8]
```diff
@@ -1,4 +1,5 @@
...
+Copyright (c) 2014-2017, The Monero Project
```
Comment (mrheinen) [inline LICENSE]: Please change to Copyright (c) 2014-2018, The Monero Project

File: LICENSE [hunk 9]
```diff
@@ -28,4 +29,4 @@ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 Parts of the project are originally copyright (c) 2012-2013 The Cryptonote
-developers
+developers, 2014-2017 monero developers and 2017 Sumokoin developers
```
Comment (mrheinen) [inline LICENSE]: I don't know what this originally was please consider this file holy.  Can you revert the changes in this file and then simply add the Ombre copyright in addition?

File: Makefile [hunk 10]
```diff
@@ -1,4 +1,6 @@
-# Copyright (c) 2014-2018, The Monero Project
+# Copyright (c) 2014-2017, The Monero Project
```
Comment (mrheinen) [inline Makefile]: Copyright (c) 2014-2018, The Monero Project

File: Makefile [hunk 11]
```diff
@@ -1,4 +1,6 @@
...
+# Copyright (c) 2017, SUMOKOIN
```
Comment (mrheinen) [inline Makefile]: Copyright (c) 2017, SUMOKOIN  can be removed here

**Review by mrheinen — COMMENTED** [review]
Only some small comments. Thanks for the change!

File: CMakeLists.txt (hunk 2 above)
Comment (yoosofan) [reply CMakeLists.txt]: Sorry based on this comment of yours I changed them to 2017.

**Comment by yoosofan** [comment]
Thanks for review. Edited and pushed.

**Comment by selson1** [comment]
how to work your wallet ombre?  
I hope positive results are taken.

**Comment by yoosofan** [comment]
> how to work your wallet ombre?
> I hope positive results are taken.

Whenever @SadBatman somehow confirms my simple changes or telling more of his sightful review, I would spend more time on the project. Hope you and all other respectful Ombre hodlers  and community members understand why am I doing like this.

**Comment by mrheinen** [comment]
I don't understand your comments: please explain.

I'll merge the PR today/tomorrow.

**Comment by yoosofan** [comment]
Thanks for your comment. But I do not know what should I explain more. 
For short: please do not over think about my simple sentences, I will work as good as I can. My old fuel engine needs some kind of confirmation to continue it is not on auto pilot.
I would play as long as other people let me play. If they don't want me to play or my play is so bad, then it is better to stop playing. I use these kind of advices most of my life while I do know it is against most enthusiastic advice comes from semi psychological books and advisors who always invite people to play no matter what. They expect people to have eternal internal energy besides what would happen in the road to become successful, my mode is a little different. It is better I get back to work because I get some confirmation by your comment and my engine get some old fashion fuel. Sorry for lengthy answer, I thought you need more explanation.

**Comment by selson1** [comment]
> Yorumun için teşekkürler. Ama ne daha fazla açıklamalıyım bilmiyorum. 
> Kısacası: lütfen basit cümlelerimi düşünmeyin, elimden geldiğince iyi çalışacağım. Eski yakıt motorumun devam etmesi için bir tür onay gerekiyor. Otomatik pilotta değil.
> Diğer insanlar oynamama izin verdiği sürece oynayacağım. Eğer oynamayı istemiyorlarsa veya oyunum çok kötü ise, o zaman oynamaktan vazgeçmek daha iyidir. Hayatımın çoğunu bu tür tavsiyelerde kullanırım, bildiğim en heyecanlı tavsiyeye karşı olduğunu biliyorum, her zaman insanı ne olursa olsun oynamaya davet eden yarı psikolojik kitaplar ve danışmanlardan gelir. İnsanların, yolun başarılı olabilmesinin yanı sıra, ebedi iç enerjiye sahip olmalarını beklerler, benim modum biraz farklıdır. İşe geri dönersem daha iyi olur çünkü yorumunuzla ilgili bir onay alırım ve motorum eski moda yakıt alır. Uzun cevap verdiğim için üzgünüm, daha fazla açıklamaya ihtiyacın olduğunu düşündüm.

Thanks for your explanation. Good luck.

**Comment by selson1** [comment]
Thanks for your explanation. Good luck.
